turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -19,13 +19,15 @@ if [ -z "$package_manager" ]; then
   exit 1
 fi
 
+echo "node --version: $(node --version)"
+
 # Use the right command for each package manager
 if [ "$package_manager" == "npm" ]; then
-  package_manager_command="npm install"
+  package_manager_command="node ../../../packages/turbo-workspaces/dist/cli.js convert . npm --ignore-unchanged-package-manager && npm install"
```

Comment:
To answer your question, no, I do not love that I've done this `node ../../` at all, but I'm not thinking of a better way at the moment.

Reply:
Not that it's that much different, but we do [this](https://github.com/vercel/turborepo/blob/main/turborepo-tests/helpers/setup_integration_test.sh#L31) in the integration test setup.